src/zulip.rs

```diff
@@ -339,9 +358,32 @@ async fn workqueue_commands(
                 }
             )
         }
-        "help" => r"work show => show your assigned PRs
-work set-pr-limit <number>|unlimited => set the maximum number of PRs you can be assigned to"
-            .to_string(),
+        "set-rotation-mode" => {
+            let rotation_mode = match words.next() {
+                Some(value) => {
+                    if words.next().is_some() {
+                        anyhow::bail!("Too many parameters.");
+                    }
+                    match value {
+                        "on" => RotationMode::OnRotation,
+                        "off" => RotationMode::OffRotation,
+                        _ => anyhow::bail!("Unknown rotation mode {value}. Use `on` or `off`."),
+                    }
+                }
+                None => anyhow::bail!("Missing parameter."),
+            };
+            upsert_review_prefs(
+                &db_client,
+                user,
+                review_prefs.and_then(|p| p.max_assigned_prs.map(|v| v as u32)),
+                rotation_mode,
+            )
+            .await
+            .context("Error occurred while setting review preferences.")?;
+            tracing::info!("Setting rotation mode `{gh_username}` to {rotation_mode:?}");
+            format!("Rotation mode set to {rotation_mode:?}")
```

Comment:
Seems a bit inconsistent to use the debug representation here while we use a "display" print in the `show` command.

```suggestion
            format!("Rotation mode set to {}", match rotation_mode {
                RotationMode::OnRotation => "on rotation",
                RotationMode::OffRotation => "off rotation",
            })
```